src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -52154,14 +52154,12 @@ void M()
         [WorkItem(29970, "https://github.com/dotnet/roslyn/issues/29970")]
         public void Tuple_Assignment_01()
         {
-            // Type of (default, default) should not be <null>!. Was (object?, string?)
-            // https://github.com/dotnet/roslyn/issues/35010
             var source =
 @"class Program
 {
     static void F()
     {
-        (object? a, string) t = (default, default)/*T:<null>!*/; // 1
+        (object? a, string) t = (default(object), default(string)) /*T:(object?, string?)*/; // 1
```

Comment:
>(default(object), default(string)) [](start = 32, length = 34)

Is it necessary to change the test?

Reply:
Yes. `(default, default)` doesn't actually have a type, it's target typed from the LHS. Changing the tests allows us to check that we're correctly returning the type we expect to.

---
In reply to: [296373455](https://github.com/dotnet/roslyn/pull/36628#discussion_r296373455) [](ancestors = 296373455)

Reply:
Will add to the test where we use target typing too

---
In reply to: [296394150](https://github.com/dotnet/roslyn/pull/36628#discussion_r296394150) [](ancestors = 296394150,296373455)